mikelpardo/mobile-sales
# Add router and header component

**Description**

Add react-router and add header component

**How to test**

1. Go to: http://localhost:3001/
2. Check if the header title is displayed
